js/ui/search.js

```diff
@@ -377,7 +377,7 @@ const GridSearchResults = new Lang.Class({
     },
 
     _createResultDisplay: function(meta) {
-        return this.parent(meta) || new GridSearchResult(this.provider, meta);
```

Comment:
I'm not 100% sure now, but when I looked into this I think that would certainly be possible but I ended up discarding it because (1) that would force me to re-style things without reusing much of the stuff from eos 3.1, (2) because that way I could simply integrate the results with AppActivation by means of an `activateResult()` method in the AppSearchProvider (which I forgot to add, but let's ignore that detail for now...) and (3) because the patch looked pretty simple anyway.

I can spend more time if you want trying to do that, but with more pressing needs happening maybe that's a separate ticket that we can work on afterwards (and perhaps a good one for @GeorgesStavracas as he's way better than me at styling things :-))?